### ISSUE TITLE
Devastate

### ISSUE THREAD

**Issue by Xadras** [Looking4Group/L4G_Core#2244]
Originally reported by: **Tr1n1ty (Bitbucket: [Tr1n1ty_](https://bitbucket.org/Tr1n1ty_), GitHub: Unknown)**

---

Devastate seems to be a bit strange right now, it consists of the 2 Components, the Damage and the Sunder Armor

Right now it is possible that you damage a NPC with Devastate but the NPC parries/block/dodges the "Sunder Armor", even the Combat Log then says "TANKS Sunder Armor has been XYZ by NPC"

This shoud not happen at all, either Devastate hits, or it doesn't, it can't hit half, i'm unsure if u loose any aggro through this which woud make it huge, but i have to reliable way of testing this.

Also, even though when the Main Bug is fixed this shoud be obsolete, when the NPC avoids the Sunder Armor (but gets the Devastete dmg) the Sunder Armor debuff doesn't stack higher BUT it refreshes.

so TL;DR : Devastete seem to have 2 Parts from which the second can be avoided, which shoudn't happen

---
- Bitbucket: https://bitbucket.org/looking4group_b2tbc/looking4group/issue/2149

**Comment by AnonXS** [comment]
Partial Resist of Melee Spell Components, Same as Frostnova Root Resist -> Damage is not resisted.
